os/fs/smartfs/smartfs_mksmartfs.c

```diff
@@ -142,8 +142,8 @@ int mksmartfs(FAR const char *pathname, bool force)
 		}
 	}
 
-	/* Validate the block device is a SMART device */
-	/* Perform a low-level SMART format */
+	/* Validate the block device is a jSmartFs device */
+	/* Perform a low-level jSmartFs format */
```

Comment:
I think these should be avoided as well.
The block device is a SMART device and it stays that way. It is our Filesystem that has been enhanced logically with journaling and recovery methods.
@Taejun-Kwon your opinion please?